events/README.md

```diff
@@ -0,0 +1,472 @@
+# Events
+
+## Events object
+
+```
+{
+  'name': string,
+  'description': string,
+  'room_id':string,
+  'template_id': string,
+  'enabled': boolean,
+  'lock': boolean,
+  'region': <'in' | 'us' | 'eu' | 'auto'>,
+  'peers': [
+    "<peer_id>",
+    "<peer_id>",
+    "<peer_id>"
+   ]
+  'created_by': {
+    ref: 'User',
+  },
+  'questions': [
+    "<question_id>",
+    "<question_id>",
+    "<question_id>"
+  ],
+  'comments: [
+    "<comment_id>",
+    "<comment_id>",
+    "<comment_id>",
```

Comment:
can we call it `comment_object_id` better ?

Reply:
Yes, fixed it!